cin_validator/rules/cin2022_23/rule_8820.py

```diff
@@ -87,23 +87,22 @@ def validate(
 
     # Determine overlaps
     cin_started_after_start = (
-        df_merged["CINreferralDate_cin"] >= df_merged["CINreferralDate_cin2"]
+        df_merged["CINreferralDate_cin"] > df_merged["CINreferralDate_cin2"]
     )
     cin_started_before_end = (
-        df_merged["CINreferralDate_cin"] <= df_merged["CINclosureDate_cin2"]
+        df_merged["CINreferralDate_cin"] < df_merged["CINclosureDate_cin2"]
     ) & df_merged["CINclosureDate_cin2"].notna()
 
-    falsezero = ["False", "0"]
     cin_started_before_refdate = (
-        (df_merged["CINreferralDate_cin"] <= reference_date)
+        (df_merged["CINreferralDate_cin"] < reference_date)
         & df_merged["CINclosureDate_cin2"].isna()
-        & df_merged["ReferralNFA_cin2"].isin(falsezero)
+        & df_merged["ReferralNFA_cin2"].isin(["false", "0", 0, "False"])
```

Comment:
all non-date columns that reach the rules can be trusted to be in string format. So we don't need to check for integers. 
Using `str.lower() ` also means we can be happy with all possible capitalisations of "False". 
`column_name.str.lower().isin(...)`